### PR TITLE
First fully working version which delivers ROOT ntuples via Uproot ServiceX

### PR DESCRIPTION
**Pull request by kyungeonchoi**
First fully working version of the library for a delivery of ROOT ntuples via Uproot ServiceX.

### CODE REVIEW

File: servicex_for_trexfitter/make_histograms.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                hist_name = request['Region'] + "_" + request['Sample']
+                h = coffea.hist.Hist(hist_name, coffea.hist.Bin("var", "", hist_binning))
+                for outfile in output:
+                    columns = pq.read_table(outfile)
+                    h.fill(var=numpy.array(columns.column(request['Variable'].split(",")[0])))
+                fout[hist_name] = coffea.hist.export1d(h)
+        fout.close()
+
+                
+                ### Boost-histogram version
```
Comment (ponyisi) [inline servicex_for_trexfitter/make_histograms.py]: Suggest deleting commented code before distribution

File: servicex_for_trexfitter/read_trex_config.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
+import re
+import linecache
+import json
+
+def read_configuration(confFile: str, block_name: str):
+    """
+    Read TRExFitter configuration file and return block as python dict
+    """
+    Block = {}
```
Comment (ponyisi) [inline servicex_for_trexfitter/read_trex_config.py]: Somewhat unusual to have local variables with capitalized names

File: servicex_for_trexfitter/read_trex_config.py [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    with open( confFile ) as configFile:
+        num = 0
+        for mark,line in enumerate(configFile.readlines()):        
+            if re.search(r'\b{}\b'.format(block_name), line):
```
Comment (ponyisi) [inline servicex_for_trexfitter/read_trex_config.py:13]: In the long term: may prefer a proper parser... but as long as it works for now...

File: servicex_for_trexfitter/make_histograms.py (hunk 1 above)
Comment (kyungeonchoi) [reply servicex_for_trexfitter/make_histograms.py]: Done

File: servicex_for_trexfitter/read_trex_config.py (hunk 2 above)
Comment (kyungeonchoi) [reply servicex_for_trexfitter/read_trex_config.py]: changed

File: servicex_for_trexfitter/read_trex_config.py (hunk 3 above)
Comment (kyungeonchoi) [reply servicex_for_trexfitter/read_trex_config.py:13]: Yeah, I will create an issue for this

**Pull request merged**